features/mbedtls/targets/TARGET_STM/sha1_alt.h

```diff
@@ -0,0 +1,127 @@
+/*
+ *  sha1_alt.h SHA-1 hash
+ *******************************************************************************
+ *  Copyright (C) 2017, STMicroelectronics
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+#ifndef MBEDTLS_SHA1_ALT_H
+#define MBEDTLS_SHA1_ALT_H
+
+#if defined MBEDTLS_SHA1_ALT
+
+#include "mbedtls/platform.h"
```

Comment:
It is needed to avoid warnings that 'malloc' is declared implicitly. I move it to sha1_alt.c instead